using-mobile-extensions/adobe-target/README.md

```diff
@@ -84,271 +98,152 @@ public class TargetApp extends Application {
-
-Map<String, Object> profileParameters1 = new HashMap<>();
-profileParameters1.put("gender", "male");
-
-TargetParameters targetParameters = new TargetParameters.Builder()
-                                    .parameters(mboxParameters1)
-                                    .profileParameters(profileParameters1)
-                                    .product(targetProduct)
-                                    .order(targetOrder)
-                                    .build();
 ```
 {% endtab %}
 
 {% tab title="iOS" %}
 #### Syntax
 
 ```objectivec
-+ (nonnull instancetype) targetOrderWithId: (nonnull NSString*) orderId
-                                     total: (nullable NSNumber*) total
-                       purchasedProductIds: (nullable NSArray <NSString*>*) purchasedProductIds;
-
 + (nonnull instancetype) targetProductWithId: (nonnull NSString*) productId
-                                  categoryId: (nullable NSString*) categoryId;
-
-+ (nonnull instancetype) targetParametersWithParameters: (nullable NSDictionary*) targetParameters
-                                      profileParameters: (nullable NSDictionary*) profileParameters
-                                                product: (nullable ACPTargetProduct*) product
-                                                  order: (nullable ACPTargetOrder*) order;
+categoryId: (nullable NSString*) categoryId;
 ```
 
-#### Objective-C Example
+#### Examples
 
-```objectivec
-NSDictionary *mboxParameters = @{@"status":@"Platinum"};
-NSDictionary *profileParameters = @{@"gender":@"female"};
+Here are some examples in Objective-C and Swift:
+
+**Objective-C**
 
+
+```objectivec
 ACPTargetProduct *product = [ACPTargetProduct targetProductWithId:@"24D334" categoryId:@"Stationary"];
+```
 
-ACPTargetOrder *order = [ACPTargetOrder targetOrderWithId:@"ADCKKBC" total:@(400.50) purchasedProductIds:@[@"34", @"125"]];
+**Swift**
 
-ACPTargetParameters *targetParameters = [ACPTargetParameters targetParametersWithParameters:mboxParameters
-                                                    profileParameters:profileParameters
-                                                              product:product
-                                                                order:order];
+```swift
+let product = ACPTargetProduct(id: "24D334", categoryId: "Stationary")
 ```
 {% endtab %}
 {% endtabs %}
 
-### Using Locations Displayed API for Prefetch
 
-In some situations, you might want to want to inform Target that the corresponding location \(mbox\) has been viewed. This API sends a display notification to Target for a prefetched mbox, which helps Target record location display events.
+### Target Parameters
 
-Tip: If you are only using regular mboxes, and not prefetching any mbox content, do not call this method.
+`TargetParameters` encapsulates `mboxParameters`, `profileParameters`, `orderParameters` and `productParameters` and provides an easy way to pass these parameters in a Target request. 
```

Comment:
Please reword as follows:
"`TargetParameters` encapsulates `mboxParameters`, `profileParameters`, `orderParameters` and `productParameters` and allows you easily pass these parameters in a Target request. "